plugins/GSdx/GSdx.cpp

```diff
@@ -417,6 +417,11 @@ void GSdxApp::SetConfig(const char* entry, const char* value)
 	WritePrivateProfileString(m_section.c_str(), entry, value, m_ini.c_str());
 }
 
+int GSdxApp::GetConfig(const char* entry, int value)
```

Comment:
Well, the `GSRendererType::Undefined` default might be a good idea if you extend the default renderer stuff so it works with `_GSOpen` too. That way you don't need to re-add `GetConfig` and can avoid bypassing the defaults system as well.